terraform/infrastructure/modules/01-network/05-network.tf

```diff
@@ -0,0 +1,32 @@
+﻿module "vpc" {
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "~> 5.0"
+
+  name = var.name
+  cidr = var.vpc_cidr
+
+  azs = local.azs
+
+  create_database_subnet_group = true
+
+  # divide /16 into a public /24, one private /25, and one database /25
+  public_subnets   = [for k, v in local.azs : cidrsubnet(var.vpc_cidr, 8, k)]
+  private_subnets  = [for k, v in local.azs : cidrsubnet(var.vpc_cidr, 8, k + 128)]
+  database_subnets = [for k, v in local.azs : cidrsubnet(var.vpc_cidr, 8, k + 128 + 64)]
```

Comment:
I think the code and comment here are doing two very different things.

Assuming `var.vpc_cidr = 10.0.0.0/16`
We have:
public_subnets: `10.0.0.0/24`, `10.0.1.0/24`, ...
private_subnets: `10.0.128.0/24`, `10.0.129.0/24`, ...
database_subnets: `10.0.192.0/24`, `10.0.193.0/24`, ...

Effectively this means that both private and database subnets are /24 with the third byte of their netmask being equal to `0b10000000` and `0b11000000` respectively.